compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/RenderingUIView.uikit.kt

```diff
@@ -67,7 +66,7 @@ internal class RenderingUIView(
             }
 
             override fun retrieveInteropTransaction(): UIKitInteropTransaction =
-                interopContext.retrieve()
+                this@RenderingUIView.retrieveInteropTransaction()
```

Comment:
```suggestion
                retrieveInteropTransaction()
```

Reply:
This one will call the method itself -> infinite recursion